README.md

```diff
@@ -116,6 +116,17 @@ checkerFramework {
 The check for test targets is entirely syntactic: this option will not apply the checkers
```

Comment:
If you want to be really picky about how the markdown looks, you need to change the indentation of the text.

Compare:

```
* a

b

* c
```

which renders as:

* a

b

* c

To:

```
* a

  b

* c
```

* a

  b

* c

Just as an FYI, let's not do this in the current PR.